src/index.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react'
 import { render } from 'react-dom'
-import { createStore } from 'redux'
+import { createStore, applyMiddleware } from 'redux'
 import { Provider } from 'react-redux'
 import App from './containers/App'
 import reducer from './reducers'
 import 'todomvc-app-css/index.css'
 
-const store = createStore(reducer)
+import { createCaptureMiddleware, createPersistHandler, createRetrieveHandler, createRetrieveMiddleware, createReplayMiddleware, wrapReducer } from 'redux-vcr';
+
+const firebaseAuth = {
+  apiKey: "AIzaSyBklunZBwf0YMFbpHWVmfcu2scFAb2gq3o",
+  authDomain: "todomvc-ccb8e.firebaseapp.com",
+  databaseURL: "https://todomvc-ccb8e.firebaseio.com",
+};
+
+const persistHandler = createPersistHandler({ firebaseAuth });
+const retrieveHandler = createRetrieveHandler({ firebaseAuth });
+
+const middlewares = [
+  createCaptureMiddleware({ persistHandler }),
+  createRetrieveMiddleware({ retrieveHandler }),
+  createReplayMiddleware({ maximumDelay: 1000 }),
+];
+
+const store = createStore(
+  wrapReducer(reducer),
+  applyMiddleware.apply(this, middlewares)
+);
```

Comment:
`wrapReducer` is a higher-order reducer which allows our ReduxVCR state to live happily in tandem with your application-specific Redux state. 

It handles restoring the default application state when a new cassette is loaded, so that multiple cassettes can be played in sequence without weird things happening.